tests/python/gpu/test_gluon_model_zoo.py

```diff
@@ -33,6 +33,8 @@ def download_data():
         'http://data.mxnet.io/data/val-5k-256.rec', VAL_DATA)
 
 def test_models():
+    # We use network models without dropout for testing.
+    # TODO(zhengda) mobilenet can't pass this test even without MKLDNN.
     all_models = ['resnet18_v1', 'densenet121', 'mobilenet1.0']
```

Comment:
You can disable dropout by
```python
with autograd.record(train_mode=False):
    ...
```
Doc at https://mxnet.incubator.apache.org/versions/master/api/python/autograd/autograd.html?highlight=autograd.record#mxnet.autograd.record

Reply:
Does it run the model in the inference mode?